session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -75,6 +75,7 @@
   private long timeout; //ms
   private static int FINAL_RETRY = RETRY - 1;
   private boolean enableCompression = false;
+  private boolean enableCacheLeader = false;
```

Comment:
Where is this set?

Reply:
User can use `public Session(String host, int rpcPort, String username, String password,
      boolean isEnableCacheLeader)` interface to enable leader cache

Reply:
OK, I will add this interfance for SessionPool~